agent/src/twitter/execute-interaction.ts

```diff
@@ -64,7 +65,7 @@ export async function getTweetMessages({
     }
   } while (searchId);
 
-  return tweets.map(tweet => {
+  return await Promise.all(tweets.map(async (tweet) => {
```

Comment:
I think we should chunk and process these. Not really sure how many tweets we getting but a good practice. We can do chunks of 10 for now ot be cautious and later tweak it